src/styles/_header.scss

```
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+    transition-property: color;
+  }
+
+  &__link:hover {
```

```suggestion
  &:hover {
```
write inside &__link